iphone/cli/commands/_build.js

```diff
@@ -207,6 +207,34 @@ iOSBuilder.prototype.assertIssue = function assertIssue(issues, name) {
 };
 
 /**
+ * Retreieves the certificate information by name.
+ *
+ * @param {String} name - The cert name.
+ * @param {String} [type] - The type of cert to scan (developer or distribution).
+ * @returns {Object|null}
+ * @access private
+ */
+iOSBuilder.prototype.getCert = function getCert(name, type) {
```

Comment:
Preferring to use `getCertificate` and `certificate` all around the place, simply because it's more descriptive. But fine with this.